deploy.py

```diff
@@ -308,7 +308,7 @@ def checked_command_output(self, cmd, env=None, cwd=None):
 
   def update_rustc_if_needed(self):
     target_toolchain_fullstring = "stable"
-    with open("rust-toolchain", "r") as f:
+    with open("rust-toolchain", "r", encoding='utf-8') as f:
```

Comment:
Thanks for fixing it.
For consistency across the file (I think there might be a check in pylint or flake), please use double quotes.

Reply:
Ok, done. I missed that fact.